src/auro-header.js

```diff
@@ -63,6 +63,15 @@ export class AuroHeader extends LitElement {
         case 'xl': return `util_stackMarginXl--top util_stackMarginXl--bottom`
         case 'xxl': return `util_stackMarginXxl--top util_stackMarginXxl--bottom`
         case 'xxxl': return `util_stackMarginXxl--top util_stackMarginXxl--bottom`
```

Comment:
Shouldn't all these t-shirt size cases be removed?

If we are leaving them we should be some comments here as well that show it as deprecated but why we kept them like we did up on 19.

Reply:
@blackfalcon , could you please explain as this code was changed by you?

Reply:
if we remove that means MAJOR breaking change. the concept is deprecated and documented with this update. or 

> * @attr {String} size - Specify size of margin adjustment, either `none`, `25`, `50`, `100`, `150`, `200`, `300`, `400`, `600` or `800`. **DEPRECATED** the `xxxs` - `xxxl` options.